src/clj/game/cards/resources.clj

```diff
@@ -429,6 +429,18 @@
                              (str "add " (:title c) " to their score area and gain " (get-agenda-points state :runner c)
                                   " agenda point" (when (> (get-agenda-points state :runner c) 1) "s"))))}]}
 
+   "Find the Truth"
+   {:events {:post-runner-draw {:msg (msg "reveal that they drew: "
```

Comment:
When using `:msg`, the string is automatically prefixed with:  "<Runner name> uses <card name> to "